examples/graphbolt/rgcn/download.py

```diff
@@ -0,0 +1,686 @@
+import argparse, hashlib, os, shutil, tarfile, yaml
+import subprocess
+import urllib.request as ur
+
+import dgl.graphbolt as gb
+import numpy as np
+from tqdm import tqdm
+
+GBFACTOR = 1 << 30
+
+
+def build_yaml_helper(path, dataset_size, in_memory=True):
+    """The stirng to build the yaml file. (Still need modification)"""
+    data = {
+        "graph": {
+            "nodes": [
+                {"num": num_nodes[dataset_size]["paper"], "type": "paper"},
+                {"num": num_nodes[dataset_size]["author"], "type": "author"},
+                {
+                    "num": num_nodes[dataset_size]["institute"],
+                    "type": "institution",
+                },
+                {
+                    "num": num_nodes[dataset_size]["fos"],
+                    "type": "field_of_study",
+                },
+            ],
+            "edges": [
+                {
+                    "path": "edges/author__affiliated_to__institute.npy",
+                    "type": "author:affiliated_to:institution",
+                    "format": "numpy",
+                },
+                {
+                    "path": "edges/paper__written_by__author.npy",
+                    "type": "paper:written_by:author",
+                    "format": "numpy",
+                },
+                {
+                    "path": "edges/paper__cites__paper.npy",
+                    "type": "paper:cites:paper",
+                    "format": "numpy",
+                },
+                {
+                    "path": "edges/paper__topic__fos.npy",
+                    "type": "paper:has_topic:field_of_study",
+                    "format": "numpy",
+                },
+            ],
+        },
+        "tasks": [
+            {
+                "num_classes": 19,
+                "validation_set": [
+                    {
+                        "data": [
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/validation_indices_19.npy",
+                                "name": "seeds",
+                                "format": "numpy",
+                            },
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/validation_labels_19.npy",
+                                "name": "labels",
+                                "format": "numpy",
+                            },
+                        ],
+                        "type": "paper",
+                    }
+                ],
+                "name": "node_classification_19",
```

Comment:
If the other is going to be 2K, then maybe we don't need 19 suffix here.
```suggestion
                "name": "node_classification",
```